geopandas/plotting.py

```diff
@@ -587,6 +587,27 @@ def plot_dataframe(
     -------
     ax : matplotlib axes instance
 
+    Examples
+    --------
+    >>> df = geopandas.read_file(geopandas.datasets.get_path("naturalearth_lowres"))
+    >>> df.head()  # doctest: +SKIP
+        pop_est      continent                      name iso_a3  \
+gdp_md_est                                           geometry
+    0     920938        Oceania                      Fiji    FJI      8374.0  MULTIPOLY\
+GON (((180.00000 -16.06713, 180.00000...
+    1   53950935         Africa                  Tanzania    TZA    150600.0  POLYGON (\
+(33.90371 -0.95000, 34.07262 -1.05982...
+    2     603253         Africa                 W. Sahara    ESH       906.5  POLYGON (\
+(-8.66559 27.65643, -8.66512 27.58948...
+    3   35623680  North America                    Canada    CAN   1674000.0  MULTIPOLY\
+GON (((-122.84000 49.00000, -122.9742...
+    4  326625791  North America  United States of America    USA  18560000.0  MULTIPOLY\
+GON (((-122.84000 49.00000, -120.0000...
+
+    >>> df.plot("pop_est", cmap="Blues")  # doctest: +SKIP
+
+    See the User Guide page :doc:`../../user_guide/mapping` for details.
```

Comment:
The problem in general with using links like this in docstrings, is that those are not helpful when looking at the help interactively (eg in a ipython console or notebook). For this reason, we generally use plain links in the pandas docstrings (which has its own disadvantages, of course!)

Reply:
Makes sense but where should I point it with our messy docs situation? geopandas.org? readthedocs/stable? readthedocs/latest?

Reply:
> Makes sense but where should I point it with our messy docs situation? geopandas.org? readthedocs/stable? readthedocs/latest?

Hmm, yes, that's also a good point .. (maybe leave it as is for now, then?)